### DISCUSSION TITLE
HB Revision Module

### DISCUSSION THREAD

**Discussion by giscus** [hbstack/site#788] [General]
# HB Revision Module

Describe page with extra info, such as the create date, modified date and so on.

https://hbstack.dev/modules/components/revision/

**Comment by pleasantone** [comment]
I think this needs a documentation update, also...
```WARN  [github.com/hbstack/revision] unsupported repo service: %!s(<nil>)```

Looks like GitHub/gitlab now needs specification instead of just looking at GitInfo?

**Reply by razonyang** [reply]
Thanks reminding, fixed and updated docs.

> Looks like GitHub/gitlab now needs specification instead of just looking at GitInfo?

If you want to display the repo actions (edit/view/history) and contributors (via calling API), then yes. Only `github` are supported so far.

**Reply by razonyang** [reply]
Just setup the repo's actions and contributors on this docs site, you can preview it below the doc's content.
It's typically used by open authoring/cooperating site, to show the credit for authors and editors, such as open source project's documentation site.

Those newer features are also an alternative of content panel.

**Reply by pleasantone** [reply]
Yep, nice work. I was using it just to show the last revision in case an article was updated.
